sdk/agrifood/azure-agrifood-farming/README.md

```diff
@@ -245,15 +271,25 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
 This project has adopted the [Microsoft Open Source Code of Conduct][code_of_conduct]. For more information see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
 
 <!-- LINKS -->
-
-[azure_subscription]: https://azure.microsoft.com/free/
-[pip]: https://pypi.org/project/pip/
+[api_docs]: https://azuresdkdocs.blob.core.windows.net/$web/python/azure-agrifood-farming/latest/index.html
 [authenticate_with_token]: https://docs.microsoft.com/azure/cognitive-services/authentication?tabs=powershell#authenticate-with-an-authentication-token
 [azure_identity_credentials]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity#credentials
 [azure_identity_pip]: https://pypi.org/project/azure-identity/
-[default_azure_credential]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity#defaultazurecredential
-[python_logging]: https://docs.python.org/3.5/library/logging.html
+[azure_subscription]: https://azure.microsoft.com/free/
+[change_log]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/agrifood/azure-agrifood-farming/CHANGELOG.md
 [cla]: https://cla.microsoft.com
-[code_of_conduct]: https://opensource.microsoft.com/codeofconduct/
-[coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
 [coc_contact]: mailto:opencode@microsoft.com
+[coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
+[code_of_conduct]: https://opensource.microsoft.com/codeofconduct/
+[default_azure_credential]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity#defaultazurecredential/
+[farm_hierarchy]: https://aka.ms/FarmBeatsFarmHierarchyDocs
```

Comment:
Lets have all the aka.ms links point to a single page that is already live. Once you have the individual docs up, you can go change the aka.ms link